demo/api/src/brevo-contact/dto/brevo-contact-attributes.ts

```diff
@@ -0,0 +1,19 @@
+import { IsUndefinable } from "@comet/cms-api";
+import { Field, InputType, ObjectType } from "@nestjs/graphql";
+import { IsNotEmpty, IsString } from "class-validator";
+
+@ObjectType()
+@InputType("BrevoContactAttributesInput")
+export class BrevoContactAttributes {
+    @IsUndefinable()
+    @IsNotEmpty()
+    @IsString()
+    @Field()
+    LASTNAME: string;
+
+    @IsUndefinable()
+    @IsNotEmpty()
+    @IsString()
+    @Field()
+    FIRSTNAME: string;
```

Comment:
here as well

Reply:
I removed the decorator in [20ec2d4](https://github.com/vivid-planet/comet-brevo-module/pull/6/commits/20ec2d4bbc50ade332021d9b17b38be40b412a35)